config/qa.env

```diff
@@ -0,0 +1,15 @@
+NODE_ENV=production
+OAUTH_CLIENT_ID=AQECAHh7ea2tyZ6phZgT4B9BDKwguhlFtRC6hgt+7HbmeFsrsgAAAHMwcQYJKoZIhvcNAQcGoGQwYgIBADBdBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDGjgjo/vQ4NAC/lq9gIBEIAwI7RrzwumIX3u0I0QD4LO9z8uhtgIuU75+FH5XeQl0xE42HIm2n8Mn1rkXry6TR0N
+OAUTH_PROVIDER_URL=https://isso.nypl.org/oauth/token
+NYPL_DATA_API_URL=https://platform.nypl.org/api/v0.1/
```

Comment:
This should also point to https://dev-platform.nypl.org/api/v0.1/

Reply:
you mean `qa-platform.nypl..` right?

Reply:
Yes -- am still living in "we don't have qa set up land."